components/encumbered/lms/lms.proto-objects

```diff
@@ -0,0 +1,2582 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2018 <contributor>
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=opt/lms-$(COMPONENT_VERSION)/Bin/MSWin32-x86-multi-thread/faad.exe
+file path=opt/lms-$(COMPONENT_VERSION)/Bin/MSWin32-x86-multi-thread/flac.exe
+file path=opt/lms-$(COMPONENT_VERSION)/Bin/MSWin32-x86-multi-thread/grant.exe
+file path=opt/lms-$(COMPONENT_VERSION)/Bin/MSWin32-x86-multi-thread/mac.exe
+file path=opt/lms-$(COMPONENT_VERSION)/Bin/MSWin32-x86-multi-thread/mppdec.exe
+file path=opt/lms-$(COMPONENT_VERSION)/Bin/MSWin32-x86-multi-thread/sls.exe
+file path=opt/lms-$(COMPONENT_VERSION)/Bin/MSWin32-x86-multi-thread/socketwrapper.exe
+file path=opt/lms-$(COMPONENT_VERSION)/Bin/MSWin32-x86-multi-thread/sox.exe
+file path=opt/lms-$(COMPONENT_VERSION)/Bin/MSWin32-x86-multi-thread/wmadec.exe
```

Comment:
Why do we ship non-native binaries?

Reply:
This is an error. They should not be there and will be removed.

Reply:
All that stuff came from the downloaded component tarball. I set a find-and-delete for removing all non-native stuff.